addons/ui/org.openhab.ui.cometvisu/about_files/ANOTHER-REST-CLIENT_LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
I just did it like it's done in `org.eclipse.smarthome.ui.paper`, so I thought this would be the preferred way. No problem will change that.